src/core/js/models/adaptModel.js

```diff
@@ -555,38 +558,32 @@ define([
         this._hasSiblingsAndSelf = true;
       }
 
-      var siblingsCollection = new Backbone.Collection(siblings);
+      const siblingsCollection = new Backbone.Collection(siblings);
       this.set('_siblings', siblingsCollection);
       return siblingsCollection;
-    },
-
-    setOnChildren: function (key, value, options) {
+    }
 
-      var args = arguments;
+    setOnChildren(key, value, options) {
 
-      this.set.apply(this, args);
+      this.set(...arguments);
 
       if (!this._children) return;
 
-      var children = this.getChildren();
-      var models = children.models;
-      for (var i = 0, len = models.length; i < len; i++) {
-        var child = models[i];
-        child.setOnChildren.apply(child, args);
-      }
+      const children = this.getChildren();
+      children.models.forEach(child => child.setOnChildren(...arguments));
 
-    },
+    }
 
     /**
      * @deprecated since v3.2.3 - please use `model.set('_isOptional', value)` instead
      */
-    setOptional: function(value) {
-      Adapt.log.warn("DEPRECATED - Use model.set('_isOptional', value) as setOptional() may be removed in the future");
+    setOptional(value) {
+      Adapt.log.warn('DEPRECATED - Use model.set(\'_isOptional\', value) as setOptional() may be removed in the future');
```

Comment:
Suggest template literal to irradicate need for escaping:
```suggestion
      Adapt.log.warn(`DEPRECATED - Use model.set('_isOptional', value) as setOptional() may be removed in the future`);
```